test/integration/dashboardTest/submitEditLocationTest.js

```diff
@@ -0,0 +1,465 @@
+// Third-party dependencies
```

Comment:
Moved from `serverTest.js` and then made some changes that I'll try to point out.